core/tee/tee_svc.c

```diff
@@ -43,6 +42,12 @@
 #include <trace.h>
 #include <kernel/trace_ta.h>
 #include <kernel/chip_services.h>
+#include <kernel/misc.h>
+
+#define MAX_BASE64_BUFFER_SIZE 120
+
+extern int base64_encode(const unsigned char *in,  unsigned long len,
```

Comment:
The "ext.tee.arm.genericTimerInfo" property data is returned as a binary block, and according to  TEE internal core API spec. v1.1 section 4.4, the binary block data is converted to base64 encoded string. So I need to encode the data to base64 format before returning to user. For now, OPTEE OS did not use the base64 function in kernel mode, I think we may need to support base64 in crypto abstraction layer(struct crypto_ops), so that we can avoid directly call the base64_encode() provided by libtomcrypt.